src/bin/tree-sitter-graph/main.rs

```diff
@@ -116,8 +136,7 @@ fn main() -> Result<()> {
     }
 
     let mut functions = Functions::stdlib();
-    let globals = Variables::new();
-    let mut config = ExecutionConfig::new(&mut functions, &globals).lazy(lazy);
+    let mut config = ExecutionConfig::new(&mut functions, &globals_).lazy(lazy);
```

Comment:
This is the key to how this all works: instead of passing in an empty map of `Variables`, we pass in the one we built above instead. Really quite tidy, that.